app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/onboarding/change_gg_account.scala.html

```diff
@@ -21,28 +21,34 @@
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes
 
 @this(
-       mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
-       pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
-       backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+  layout: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.Layout
 )
 
-@(backLink: Call)(implicit request: RequestWithSubscriptionReady[_], messages: Messages, appConfig: ViewConfig)
+@(
+  backLink: Call
+)(implicit
+  request: RequestWithSubscriptionReady[_],
+  messages: Messages,
+  appConfig: ViewConfig
+)
 
 @title = @{messages("changeGGAccount.title")}
 
-@mainTemplate(title = title, userType = request.sessionData.userType) {
-
-   @backLinkComponent(backLink)
-
-   @pageHeading(title)
+@layout(
+  title,
+  backLinkUrl = Some(backLink.url)
+) {
+  <h1 class="govuk-heading-xl">
+    @title
+  </h1>
 
-   <p>
+  <p class="govuk-body">
     @messages("changeGGAccount.p")
-   </p>
-   <p>
-    <a href="@{appConfig.signOutUrl}?continue=@{routes.StartController.start().url}" class="button" role="button">
+  </p>
+
+  <p class="govuk-body">
+    <a href="@{appConfig.signOutUrl}?continue=@{routes.StartController.start().url}" class="govuk-link" role="button">
```

Comment:
to preserve the presentation as a button this should be a GovukButton with an href value - previously there was class "button" on the link which turned it into a green button

Reply:
done